app/assets/javascripts/accordion-with-descriptions.js

```diff
@@ -121,9 +121,12 @@
         if (isOpenSubsections) {
           var $openSubsections = $('.subsection--is-open');
           $openSubsections.each(function(index) {
-            var subsectionOpenContentId = $(this).find('.subsection__content').attr('id');
-            sessionStorage.setItem( GOVUKServiceManualTopic+subsectionOpenContentId , 'Opened');
-          });
+            $(this)
+              .find('.subsection__content')
+              .each(function(i) {
+                sessionStorage.setItem( GOVUKServiceManualTopic + $(this).attr('id') , 'Opened');
```

Comment:
Is `GOVUKServiceManualTopic` some boiler plate, can we change it to `RegisterPageSection` or whatever it represents?

Reply:
I've changed the variable names - but I want to avoid changing the accordion javascript any more. There is other work that could be done on it to make the accordion more effective, but I want to know that an accordion will be used in the future and to see what the new accordion does before jumping into this in any more depth.